plugins/networklayer_tcp.c

```diff
@@ -426,8 +427,10 @@ ServerNetworkLayerTCP_getJobs(UA_ServerNetworkLayer *nl, UA_Job **jobs, UA_UInt1
     size_t j = 0;
     UA_ByteString buf = UA_BYTESTRING_NULL;
     for(size_t i = 0; i < layer->mappingsSize && j < (size_t)resultsize; i++) {
-        if(!UA_fd_isset(layer->mappings[i].sockfd, &fdset))
-            continue;
+        if(UA_fd_isset(layer->mappings[i].sockfd, &errset)) {}
```

Comment:
@NetoLe Can you explain line 430?
If the errset is set for the socket, we want to continue (and close the socket)?
I would fold line 430 and line 431 into a single if. The empty brackets can be overseen.